swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
 class Item:
-    pass
+    
+    def __init__(self, category = "",condition = 0):
+        #self.category = ""
+        self.category = category 
+        self.condition = condition
+
+    def __str__(self):
+        return("Hello World!")
+
+    def condition_description(self, condition = 0):
+        if self.condition == 0.0:
+            return("Yikes on bikes")
```

Comment:
this works! I think you'll see single values returned without the parentheses:

```suggestion
            return "Yikes on bikes"
```